torch_geometric/loader/utils.py

```
@@ -224,3 +228,22 @@ def filter_feature_store(
         data[attr.group_name][attr.attr_name] = tensors[i]
 
     return data
+
+
+def has_edges(data: Union[Data, HeteroData]) -> bool:
+    """Returns :obj:`True` if :obj:`data` has attribute
+    `edge_index`, `adj_t` or `adj`."""
+    def edge_present(edge_store: Union[Data, EdgeStorage]):
+        return ((hasattr(edge_store, 'edge_index')
+                 and edge_store.edge_index is not None) or
+                (hasattr(edge_store, 'adj_t') and edge_store.adj_t is not None)
+                or (hasattr(edge_store, 'adj') and edge_store.adj is not None))
+
+    if isinstance(data, Data):
+        return True if edge_present(data) else False
+    else:
+        for store in data.edge_stores:
+            if edge_present(store):
+                return True
+        return False
+    # TODO: check if graph store has edges
```

should we raise an error here for now?

---

Done.